core-rust/core-api-server/src/core_api/conversions/numerics.rs

```diff
@@ -150,37 +150,30 @@ pub fn to_unix_timestamp_ms(time: std::time::SystemTime) -> Result<i64, MappingE
     })
 }
 
-pub fn to_api_instant(instant: &Instant) -> Result<models::Instant, MappingError> {
-    to_api_instant_from_safe_timestamp(instant.seconds_since_unix_epoch.checked_mul(1000).ok_or(
-        MappingError::IntegerError {
-            message: "Timestamp must be representable as millis in i64".to_owned(),
-        },
-    )?)
+pub fn to_api_scrypto_instant(instant: &Instant) -> Result<models::ScryptoInstant, MappingError> {
+    let timestamp_seconds = instant.seconds_since_unix_epoch;
+    let date_time = NaiveDateTime::from_timestamp_opt(timestamp_seconds, 0);
```

Comment:
Can we add some unit tests for the extremes of RFC 3339 / ISO 8601 support (both inside and outside of the supported range)?